app/mailers/update_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateMailer < ApplicationMailer
```

Comment:
I like that you went ahead and added a new mailer, but `#settings_email` could have been built into the UserMailer since it's mailing the user